Update Readme.md (#9)
riscv/riscv-edk2-platforms: Platform/RISC-V/PlatformPkg/Readme.md

There are two packages to support RISC-V edk2 platforms:
- `edk2-platforms/Silicon/RISC-V/ProcessorPkg/RiscVProcessorPkg.dec`
- `edk2-platforms/Platform/RISC-V/PlatformPkg/RiscVPlatformPkg.dec`

`RiscVPlatformPkg` currently provides the generic SEC driver for all RISC-V platforms, and some platform level libraries.
`RiscVProcessorPkg` currently provides RISC-V processor related libraries, PEI modules, DXE drivers and industrial
standard header files.

## EDK2 RISC-V Platform Package
RISC-V platform package provides the common modules for RISC-V platforms. RISC-V platform vendors could include
RiscPlatformPkg.dec to use the common drivers, libraries, definitions, PCDs and etc. for the
RISC-V platform development.

### Download the Source Code ###
```
git clone https://github.com/tianocore/edk2.git
git clone https://github.com/tianocore/edk2-platforms.git

```

You have to follow the build steps for EDK2 (https://github.com/tianocore/tianocore.github.io/wiki/Getting-Started-with-EDK-II)
and additionally set an environment variable to point to your RISC-V toolchain binaries for building RISC-V
platforms,
```

> # e.g. If the tollchain binaries is under /riscv-gnu-toolchain-binaries/bin

Reviewer: ```suggestion
# e.g. If the toolchain binaries are under /riscv-gnu-toolchain-binaries/bin

```

Author: done
